src/neptune_prophet/impl/__init__.py

```diff
@@ -59,14 +63,16 @@
 )
 from prophet.serialize import model_to_json
 
+from neptune_prophet.impl.version import __version__
+
 
 def create_summary(
     model: Prophet,
     df: Optional[pd.DataFrame] = None,
     fcst: Optional[pd.DataFrame] = None,
     log_charts: bool = True,
     log_interactive: bool = False,
-) -> Dict[str, Any]:
+) -> StringifyValue | Mapping | list | tuple:
```

Comment:
Was this incorrectly annotated previously? 

Cause based on previous only `Mapping` makes sense.

Reply:
This is the signature of `stringify_unsupported`